config.main.json

```diff
@@ -0,0 +1,119 @@
+{
+  "domains": [
+    {
+      "id": 1,
+      "chainId": 1,
+      "name": "ethereum",
+      "type": "evm",
+      "bridge": "0x4D878E8Fb90178588Cda4cf1DCcdC9a6d2757089",
+      "handlers": [
+        {
+          "type": "erc20",
+          "address": "0xC832588193cd5ED2185daDA4A531e0B26eC5B830"
+        },
+        {
+          "type": "permissionlessGeneric",
+          "address": "0xa176F012b3bBD7D4824eB31E8707FcE64F64f29f"
+        }
+      ],
+      "nativeTokenSymbol": "eth",
+      "nativeTokenFullName": "ether",
+      "nativeTokenDecimals": 18,
+      "blockConfirmations": 5,
+      "startBlock": 17471012,
+      "feeRouter": "0xC47468aeae431f5D0B7DA50F9f5D8a6c0eca4789",
+      "feeHandlers": [
+        {
+          "address": "0xe43F8245249d7fAF46408723Ab36D071dD85D7BB",
+          "type": "basic"
+        }
+      ],
+      "resources": [
+        {
+          "resourceId": "0x0000000000000000000000000000000000000000000000000000000000000001",
+          "type": "fungible",
+          "address": "0x6c5bA91642F10282b576d91922Ae6448C9d52f4E",
+          "symbol": "PHA",
+          "decimals": 18
+        },
+        {
+          "resourceId": "0x0000000000000000000000000000000000000000000000000000000000000000",
+          "type": "permissionlessGeneric",
+          "address": "",
+          "symbol": "",
+          "decimals": 0
+        }
+      ]
+    },
+    {
+      "id": 2,
+      "chainId": 5232,
+      "name": "khala",
+      "type": "substrate",
+      "bridge": "",
+      "handlers": [
+        {
+          "address": "5EYCAe5jLbHcAAMKvLFSXgCTbPrLgBJusvPwfKcaKzuf5X5e",
+          "type": "erc20"
+        }
+      ],
+      "nativeTokenSymbol": "pha",
+      "nativeTokenFullName": "pha",
+      "nativeTokenDecimals": 12,
```

Comment:
PHA token is 18 decimals on Ethereum, 12 on Phala and Khala. We set this value as 18 in testnet config, if that works well then this should also be 18 instead of 12

Reply:
Wasn't sure about that, it's true for testnet but remembered that someone mentioned some problems with decimals so figured this was the problem. Will replicate testnet